packages/ombits-client/src/components/footer.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { useInterval } from 'react-use';
+import { DateTime } from 'luxon';
+
+export const Footer = () => {
+  const dt = DateTime.local();
+  const [{ date, time }, setDatetime] = useState({
+    date: dt.toLocaleString(DateTime.DATE_HUGE),
+    time: dt.toLocaleString(DateTime.TIME_SIMPLE),
+  });
+
+  useInterval(() => {
```

Comment:
@remotealex any reason for adding it over setTimeout, or we can use the hooks only inside of the hooks thats why ?

Reply:
And why we need interval here bro? Does the above operation is async ?

Reply:
We're updating the clock in the footer so we need to use interval for each second :)

Reply:
Ahh didnt noticed we have clock